src/main/resources/menus/biomes.yml

```diff
@@ -139,6 +139,15 @@ items:
       lore:
         - '&7Change your island biome to Mesa.'
 
+biome-environment:
```

Comment:
I'd assume the environment will be per button, and not per biome.
Why did you choose to do it like this?